src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorRoutingRuleObject.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.Azure.Commands.FrontDoor.Cmdlets
     /// </summary>
     [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoorRoutingRuleObject"), OutputType(typeof(PSRoutingRule))]
     public class NewAzureRmFrontDoorRoutingRuleObject : AzureFrontDoorCmdletBase
-    {        
```

Comment:
Because you are adding a ParameterSet, you will need to specify a DefaultParameterSetName in this cmdlet attribute

Reply:
Yes, fixed the static analysis issue - breaking changes and signature issues